server/app/views/task.scala.html

```diff
@@ -1,17 +1,14 @@
 @import helper._
 
 @(taskDescription: String, solution: Form[SolutionForm])(implicit msg: Messages, req: RequestHeader)
-@helper.javascriptRouter("jsRoutes")(
-    routes.javascript.TaskSolver.postSolutionAjax
-)
 @main(msg("task")) {
     <div><a href="@routes.Application.index()">@msg("home")</a></div>
     <div class="taskDescription">@taskDescription</div>
     <div>
         <div class="solutionForm">
-        @form(routes.TaskSolver.postSolution()) {
+        @form(routes.Application.stub()) {
```

Comment:
word 'stub' looks weird, but if it is only the way to make it working, then it is ok